content/cpp/concepts/user-input/terms/getline/getline.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: 'getline()'
+Description: 'Gets the user input up to a character, if specified, and stores it to a variable.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Functions'
+  - 'Strings'
+  - 'Characters'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+In C++, the `getline()` function gets the user's input up to a character and stores it in a variable. If the character, referred to as the delimiting character, is not specified, then the entire input will be stored.
+
+## Syntax
+
+The `getline()` function is part of the `<string>` header.
+
+```pseudo
+#include <string>
+
+getline(cin, string, delim)
+```
+
+- `cin` is the object in the stream, which would be the user input.
+- `string` refers to the variable the user input is set to.
+- `delim` refers to a character that the user's input is stored up to.
+
+Although not recommended, `getline()` can be called on the `cin` object as a member function like this:
+
+```pseudo
+cin.getline(string, length, delim)
+```
+
+- `length` is the expected length of the character array.
+
+## Examples
+
+```cpp
+#include <iostream>
+// Variables can be declared as strings with the following header.
```

Comment:
Should this end with a period or colon?

Reply:
@Dusch4593 oh! it wasn't regarding the header `#include <string>` but about the comment above it

Reply:
Oh I see now. Hmm maybe could incorporate this comment into the text below? Then we could end it with a period. 😄

Reply:
Sounds like a plan~